src/ipc.c

```diff
@@ -19,6 +19,27 @@
 #include "utils_common.h"
 #include "utils_log.h"
 
+umf_result_t umfPoolGetIPCHandleSize(umf_memory_pool_handle_t hPool,
+                                     size_t *size) {
+    umf_result_t ret = UMF_RESULT_SUCCESS;
+
```

Comment:
add check for null input (we should have it in all our functions)

Reply:
Done